crates/net/eth-wire/src/types/blocks.rs

```diff
@@ -68,11 +68,17 @@ impl From<Vec<H256>> for GetBlockBodies {
 
 /// The response to [`GetBlockBodies`], containing the block bodies that the peer knows about if
 /// any were found.
-#[derive_arbitrary(rlp, 1)]
+#[derive_arbitrary(rlp, 16)]
 #[derive(Clone, Debug, PartialEq, Eq, RlpEncodableWrapper, RlpDecodableWrapper, Default)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct BlockBodies(
     /// The requested block bodies, each of which should correspond to a hash in the request.
+    #[cfg_attr(
+        any(test, feature = "arbitrary"),
+        proptest(
+            strategy = "proptest::collection::vec(proptest::arbitrary::any::<BlockBody>(), 0..=20)"
+        )
+    )]
```

Comment:
I don't really understand how proptest works,
why does this need to be specified at all?
I would expect that these things would be configured when the proptest is created, or is this how we pass this information to the macro?

this looks a bit weird at first but I don't really care as long as this speeds up tests

Reply:
the issue is that we are deriving proptest::Arbitrary instead of specifying how it should create these types, so it's very opaque. by default, all vecs have a range of 0..100, so this type in particular by default will have 0..100 block bodies, each with 0..100 ommers, 0..100 transactions (which can have 0..100 access list items, which can have 0..100 storage slots in each item) and so on. as you can see, this quickly explodes.

since the test is auto-generated, we can't configure this in there.

Reply:
a bit ugly :(, but necessary